### ISSUE TITLE
If-else if chains without parens are incorrectly marked as invalid

### ISSUE THREAD

**Issue by TapGhoul** [kizeevov/slint-idea-plugin#115]
The code below:
```slint
import { LineEdit } from "std-widgets.slint";
export component TestComponent1 inherits LineEdit {
    edited(text) => {
        if text.is-empty {

        } else if text.is-empty {

        }
    }
}
```
is shown as invalid in the IDE, but is actually valid when running through the slint preview.

Worth a note, statements without `else if` do not have this issue - it's specifically if-else if-else chains.
It also seems like the `else if` keywords here aren't being highlighted as keywords, even with `()`

RustRover version: 2026.2.1
Plugin version: 1.6.1
